server/src/pages/shot/editor.js

```diff
@@ -210,8 +228,9 @@ exports.Editor = class Editor extends React.Component {
     </div>;
   }
 
-  setColor(color) {
+  setColor(color, colorName) {
     this.setState({color});
+    this.setState({colorName});
```

Comment:
Could combine with line above into `this.setState({color, colorName});`

Reply:
+1